hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java

```diff
@@ -182,8 +183,9 @@ private void validateExplicitTokenDenomination(
             @NonNull final WritableTokenStore tokenStore) {
         final var denomToken = tokenStore.get(tokenNum);
         validateTrue(denomToken != null, INVALID_TOKEN_ID_IN_CUSTOM_FEES);
-        validateTrue(isFungibleCommon(denomToken.tokenType()), CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON);
+        validateTrue(isFungibleCommon(denomToken.tokenType()), INVALID_TOKEN_ID_IN_CUSTOM_FEES);
```

Comment:
Are we sure this is correct ? As per mono- service the[ response code is ](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/submerkle/FixedFeeSpec.java#L72)`CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON` . Why do we need this change ?

Reply:
Reverted